src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -313,15 +376,30 @@ public void LoadBytes()
         [Fact]
         public void ReflectionOnlyGetAssemblies()
         {
-            Assert.Equal(Array.Empty<Assembly>(), AppDomain.CurrentDomain.ReflectionOnlyGetAssemblies());
+            RemoteInvoke(() => {
```

Comment:
The implementation is external so I can't or better I don't want to check the actual implementation but I guess this is just a read operation, why is the RemoteExecutor needed here?